grpc/bitcoin.go

```diff
@@ -0,0 +1,81 @@
+package grpc
+
+import (
+	"context"
+
+	"github.com/ledgerhq/lama-bitcoin-svc/pb/v1"
+	"github.com/ledgerhq/lama-bitcoin-svc/pkg/bitcoin"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type controller struct {
+	svc bitcoin.Service
+}
+
+func NewBitcoinController() *controller {
+	return &controller{
+		svc: bitcoin.Service{},
+	}
+}
+
+func (c *controller) ValidateAddress(
+	ctx context.Context, request *pb.ValidateAddressRequest,
+) (*pb.ValidateAddressResponse, error) {
+	chainParams, err := getBitcoinChainParams(request.ChainParams)
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, err.Error())
+	}
+
+	normalizedAddress, err := c.svc.ValidateAddress(request.Address, chainParams)
```

Comment:
`address` (or even `addr`) should be sufficient given you use `svc` for `service` (and there is no other address var in the scope)